spring-cloud-stream-metrics/src/main/resources/config/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.stream.bindings.streamMetrics.contentType=application/json
```

Comment:
We shouldn't provide an `application.properties` file, because it will be overridden by the application's `application.properties` file (only one file from the classpath is considered). Instead, we should use a file with a specific name and include it via `ConfigurationProperties` or use an `EnvironmentPostProcessor`.